packages/terra-heading/src/Heading.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+import './Heading.scss';
+
+const propTypes = {
+  /**
+   * Needs descriptions
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * The color of the text. Accepts a CSS color value.
+   */
+  color: PropTypes.string,
+  /**
+   * Sets the text to display in italics.
+   */
+  isItalic: PropTypes.bool,
+  /**
+   * Sets the text to appear visually hidden. The text will still be available to screen readers.
+   */
+  isVisuallyHidden: PropTypes.bool,
+  /**
+   * Sets the heading level. One of 1, 2, 3, 4, 5, 6.
+   */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]).isRequired,
+  /**
+   * Sets the text size. One of mini, tiny, small, medium, large, huge.
+   */
+  size: PropTypes.oneOf(['mini', 'tiny', 'small', 'medium', 'large', 'huge']),
```

Comment:
More discussion on this on the tech design: https://github.cerner.com/terra/tech-designs/pull/31#discussion_r381825